### DISCUSSION TITLE
Site Redesign

### DISCUSSION THREAD

**Discussion by robmoffat** [risk-first/website#75] [Team Posts]
I've just given [Risk-First](https://riskfirst.org) a bit of a face-lift, mainly because of how terrible it looked on mobile.

I've used the [Mediumish](https://wowthemesnet.github.io/mediumish-theme-jekyll/about) template (you can see why they called it that) and created a whole new set of post-thumbnails (something I didn't have before)

A lot of the work was in trying to design something that wasn't completely confusing within the confines of the template.  I've settled on three categories:  Introduction, Risks and Practices, which have ordered collections of articles in them.

Plus, the front-page contains a list of "getting started" style articles, plus all the latest work.

Feedback gratefully appreciated!

**Comment by hovancik** [comment]
logo is a bit unreadable 

![image](https://user-images.githubusercontent.com/4745859/61770512-b8281600-aded-11e9-9b49-dfae12f809b7.png)

I am also not seeing any favicon

![image](https://user-images.githubusercontent.com/4745859/61770527-c5450500-aded-11e9-8d56-d64f76571fa1.png)

I really don't like links opening in new tab ;] 

![image](https://user-images.githubusercontent.com/4745859/61770607-f6bdd080-aded-11e9-94fa-ea10cab0118d.png)

otherwise 👍

**Comment by robmoffat** [comment]
I’ve made the logo a bit bigger, and I’m with you on the new tabs.  Should be fixed now. Also favicon done.  Wow those got complicated didn’t they?  About 10 lines of code to do it properly now :/

**Comment by hovancik** [comment]
Looks good on mobile as well :)

**Comment by andreadellacorte** [comment]
I find confusing that there are 6 "start here" links - Ideally it would be just one :)

**Comment by iliaskarim** [comment]
It looks really good Rob.

**Comment by DianeHolmes** [comment]
Love this, Rob!  This is a really good way to approach the content. :-)
ᐧ

**Comment by robmoffat** [comment]
> I find confusing that there are 6 "start here" links - Ideally it would be just one :)

Yes, it's a really good point.  This website refactor is the first step in making things more approachable.. I'll be working on this soon!

Really appreciate your input!

**Comment by andreadellacorte** [comment]
Some more pieces of feedback:

- The font is too large; being the website a sort of reference material for the approach, being able to see large parts of the content at once is important. I'd recommend reducing the font by 33% of reducing the whitespace.
- The sidebar, which was useful in providing a mind-map of the content and structure to the browsing, is gone - I don't think the homepage, as it is right now, provides the same ease of browsing. As it is right now, I find it difficult to know "where I am" in the content (am I browsing a risk type, the method, a principle, I'm not sure)
- In a way, I think the move to this format (and possibly the removal of the sidebar) makes me think I am browsing a series of disjointed posts; in that case, I'd expect each post to provide me everything I need, or to tell me "what to read first".
  - Imagine the case where I send this link to one of my colleagues https://riskfirst.org/De-Risking, to get an idea of what the page is about they'd have to click on the top to go to the https://riskfirst.org/Introduction but the introduction is actually not a great place to start... the https://riskfirst.org/Quick-Summary might be better (but it would require them to scroll all the way down through the introduction, and then click on the back arrows)
- There are links to the following and preceding posts at the bottom, but these require quite a lot of scrolling to get to, and some of the pages are very long.
- The photo and name at the top of each post should move to the bottom; it contributes to the feeling that each blog post is disjointed and that I'm browsing the opinions of the author, not reference material.

Very respectfully and with lots of love for the approach 🙏,
Andrea

**Comment by andreadellacorte** [comment]
> Some more pieces of feedback:
> 
> The font is too large; being the website a sort of reference material for the approach, being able to see large parts of the content at once is important. I'd recommend reducing the font by 33% of reducing the whitespace.
> The sidebar, which was useful in providing a mind-map of the content and structure to the browsing, is gone - I don't think the homepage, as it is right now, provides the same ease of browsing. As it is right now, I find it difficult to know "where I am" in the content (am I browsing a risk type, the method, a principle, I'm not sure)
> In a way, I think the move to this format (and possibly the removal of the sidebar) makes me think I am browsing a series of disjointed posts; in that case, I'd expect each post to provide me everything I need, or to tell me "what to read first".
> Imagine the case where I send this link to one of my colleagues https://riskfirst.org/De-Risking, to get an idea of what the page is about they'd have to click on the top to go to the https://riskfirst.org/Introduction but the introduction is actually not a great place to start... the https://riskfirst.org/Quick-Summary might be better (but it would require them to scroll all the way down through the introduction, and then click on the back arrows)
> There are links to the following and preceding posts at the bottom, but these require quite a lot of scrolling to get to, and some of the pages are very long.
> The photo and name at the top of each post should move to the bottom; it contributes to the feeling that each blog post is disjointed and that I'm browsing the opinions of the author, not reference material.
> Very respectfully and with lots of love for the approach 🙏,
> Andrea

I'd consider this website as a good example of reference material https://less.works/less/framework/why-less.html and this one as a less good one https://www.scaledagileframework.com/pi-objectives/ (as I need to continuously browse back to the homepage to click through the material I need).

**Comment by robmoffat** [comment]
thanks for your thoughts!  Much appreciated.  You're spot on that it is still confusing from the introductory perspective, and yes, I'll put some thought into your other ideas - maybe something like the sidebar needs to come back

I'm off on holiday today but I'll take a good look when I get back.
